src/main/java/com/couchbase/lite/replicator/Replication.java

```diff
@@ -641,16 +634,16 @@ public void setFilter(String filterName) {
      * Sets the documents to specify as part of the replication.
      */
     @InterfaceAudience.Public
-    public void setDocIds(List<String> docIds) {
+    public void setDocumentIDs(List<String> docIds) {
```

Comment:
setDocIds is public API, and it is documented with 
```
public List<String> getDocIds()
public void setDocIds(List<String> docIds)
```
http://developer.couchbase.com/documentation/mobile/1.1.0/develop/references/couchbase-lite/couchbase-lite/replication/replication/index.html#string-docids--get-set-

This method signature should not be changed.